dist/js/script.js

```diff
@@ -0,0 +1,360 @@
+        keys: [],
+        comment: null,
+    },
+
+    properties: {
+        capsLock: false,
+        language: true,
+        value: ''
+    },
+
+    init() {
+        //create textarea
+
+        this.elements.textarea = document.createElement('textarea');
+        this.elements.textarea.classList = 'textarea';
+        this.elements.textarea.setAttribute('autofocus', true);
+        document.body.appendChild(this.elements.textarea);
+        this.elements.textarea.value = localStorage.getItem('text');
+        
+        //create keyboard
+        this.elements.keyboard = document.createElement('div');
+        this.elements.keyboardKeys = document.createElement('div');
+        this.elements.keyboard.classList = 'keyboard';
+        this.elements.keyboardKeys.classList = 'keyboard__keys';
+        this.elements.keyboard.appendChild(this.elements.keyboardKeys);
+        document.body.appendChild(this.elements.keyboard);
+        this.elements.keys = this.elements.keyboardKeys.querySelectorAll('.key');
+        this.elements.keyboardKeys.appendChild(this.createKeys());
+
+        //add handlers
+
+        this.addData();
+        this.pressKey();
+        this.pressCaps();
+
+        //create comments
+        this.elements.comment = document.createElement('div');
+        this.elements.comment.classList = 'comment';
+        document.body.appendChild(this.elements.comment);
+        this.elements.comment.innerHTML = `<p>Клавиатура создана в OC Windows
+  '<i class="fab fa-windows"></i>'</p>
+                       <p>Для переключения языка комбинация: левыe ctrl + alt</p>
+                       <i class="far fa-smile-beam"></i>`
+
+    },
+
+    createKeys() {
+        const textarea = this.elements.textarea;
+        const fragment = document.createElement('div');
+        fragment.classList.add('fragment');
+        const keyEng = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'backspace',
+            'tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', 'enter',
+            'caps', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', '"',
+            ']', 'shift-l', 'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.',
+            'arrow-up', 'shift-r', 'ctrl-l', 'wind', 'alt-l', 'space', 'alt-r', 'arrow-l', 'arrow-down', 'arrow-r', 'ctrl-r'
+        ];
+
+        const keyRu = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'backspace',
+            'tab', 'й', 'ц', 'у', 'к', 'е', 'н', 'г', 'ш', 'щ', 'з', 'х', 'enter',
+            'caps', 'ф', 'ы', 'в', 'а', 'п', 'р', 'о', 'л', 'д', 'ж', 'э',
+            '/', 'shift-l', 'я', 'ч', 'с', 'м', 'и', 'т', 'ь', 'б', 'ю',
+            'arrow-up', 'shift-r', 'ctrl-l', 'wind', 'alt-l', 'space', 'alt-r', 'arrow-l', 'arrow-down', 'arrow-r', 'ctrl-r'
+        ];
+
+        localStorage.setItem('lang', this.properties.language);
+        lang = localStorage.getItem('lang');
+        let keys;
+        keys = lang == 'true' ? keyEng : keys = keyRu;
```

Comment:
- It would be better to use `en` and `ru` as values for `lang` variable
 - I do not see any sense to declare variable and right below assign value to it, this way would be better:
```js
let keys = 'values';
```

Reply:
Thanks, good to know for the future